pkg/applier/stackapplier.go

```diff
@@ -47,14 +48,15 @@ func NewStackApplier(path string, kubeClientFactory kubernetes.ClientFactory) (*
 		return nil, err
 	}
 	applier := NewApplier(path, kubeClientFactory)
-	log := logrus.WithField("component", "applier-"+applier.Name)
-	log.WithField("path", path).Debug("created stack applier")
+	logger := util.K0sLogger()
+	logger.WithField("component", "applier-"+applier.Name)
```

Comment:
This `component` field is going to be lost unless you either log or use the `Entry` that is returned from it.  You can use `WithFields()` for this.

Also, the `logger` you're passing to `StackApplier` won't have any of these fields.

See: https://github.com/sirupsen/logrus/issues/1039